### ISSUE TITLE
Can't commit after updating to 3.2.14

### ISSUE THREAD

**Issue by MalteHei** [carloscuesta/gitmoji-cli#509]
Hello @carloscuesta!

After updating to the latest version, the git commit hook fails.

Reproduce:
```sh
npm i -g gitmoji-cli@3.2.14
touch abc && git add abc
git commit
```

Downgrading to 3.2.13 "solves" this.

**OS**: Arch Linux (WSL)
**gitmoji -v**: 3.2.14
**Node version**: 15.3.0

**Error Message**:
```
TypeError: Cannot read property 'startsWith' of undefined
    at /home/hei/.npm-global/lib/node_modules/gitmoji-cli/lib/commands/commit/withHook/index.js:61:27
    at new Promise (<anonymous>)
    at cancelIfAmending (/home/hei/.npm-global/lib/node_modules/gitmoji-cli/lib/commands/commit/withHook/index.js:53:32)
    at cancelIfNeeded (/home/hei/.npm-global/lib/node_modules/gitmoji-cli/lib/commands/commit/withHook/index.js:71:30)
    at Object.commit (/home/hei/.npm-global/lib/node_modules/gitmoji-cli/lib/commands/commit/index.js:27:41)
    at Object.hook (/home/hei/.npm-global/lib/node_modules/gitmoji-cli/lib/cli.js:82:33)
    at findGitmojiCommand (/home/hei/.npm-global/lib/node_modules/gitmoji-cli/lib/utils/findGitmojiCommand.js:11:73)
    at Object.<anonymous> (/home/hei/.npm-global/lib/node_modules/gitmoji-cli/lib/cli.js:90:33)
    at Module._compile (node:internal/modules/cjs/loader:1102:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
```

**Comment by carloscuesta** [comment]
Hey @MalteHei thanks for opening an issue,

That's probably related with https://github.com/carloscuesta/gitmoji-cli/pull/507

**Comment by carloscuesta** [comment]
Already released `3.2.15` with the revert, we are going to fix that later!

Thanks!

**Comment by Jeremie-Chauvel** [comment]
Maybe git version ?
